contrib/utilities/indent

```diff
@@ -44,7 +44,7 @@ fi
 # collect a list of files and then spawn a parallel process to
 # indent all of them
 echo "--- Indenting all deal.II header and source files"
-for dir in include/deal.II/* source/* examples ; do
+for dir in include/deal.II/* source/* examples tests/*/ ; do
```

Comment:
yeah, this file go rewritten in master, i rebased already, will check that nothing is broken and `push --force`...